driver-core/src/test/resources/unified-test-format/command-monitoring/command.json

```diff
@@ -52,159 +54,25 @@
           }
         }
       ],
-      "expectLogMessages": [
+      "expectEvents": [
         {
           "client": "client",
-          "messages": [
+          "events": [
             {
-              "level": "debug",
```

Comment:
What DRIVERS ticket is this large diff associated with?

Reply:
This large diff seems our previous overlooking. The command.json file under `command-monitoring` used to be the exactly the same as the `command.json` under `command-logging`.